### ISSUE TITLE
[A11Y] [Dropdown] Drop down component should suppotr aria-required-children property to link parent-children to perform the accessibility functions

### ISSUE THREAD

**Issue by pankajkhanzode** [OfficeDev/office-ui-fabric-react#1095]
### Bug Report

- __Package version(s)__: (latest)
- __Browser and OS versions__: (all)

#### Describe the issue:

DropDown fabric component has missing aria-required-children property to link parent-children (list of items in drop down) to perform the accessibility functions.

#### Actual behavior:

Missing aria-required-children attribute

#### Expected behavior:

Support aria-required-children attribute option i.e. IDropdownOption[] that is supports.
`                        <Dropdown
                            label={Utils_String.empty}
                            options={this._getTimeDropdown()}
                            onChanged={this._handleTimeChange}
                            selectedKey={this.props.hour}
 />
`

### If applicable, please provide a codepen repro:

https://fabricreact.azurewebsites.net/fabric-react/master/#/examples/dropdown

**Comment by mdahamiwal** [comment]
@pankajkhanzode, accessibility requirements are not very clear here, is the ask here different from #1076 ? [`aria-required-children` ](https://dequeuniversity.com/rules/axe/1.1/aria-required-children) mentions that dropdown should have a textbox and listbox as its children element.

**Comment by pankajkhanzode** [comment]
@mdahamiwal - it is same issue, we can resolve as duplicate.
